data/core/about.cfg

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Looks like an accidental new-line at the start, but I suppose it can still be merged as-is?